From 1671c3e1d7b97ab21bb21f65fdeb7fab558854e1 Mon Sep 17 00:00:00 2001
From: Subrata Chatterjee <schatterjee@habana.ai>
Date: Fri, 01 Sep 2023 13:00:47 +0300
Subject: [PATCH] [SW-147246] packge: zephyr: add boot info, mbedtls for arcevt

This patch adds boot_info as module dependency for arcevent
zephyr app. arvevent app is going to use security driver api which
in turn used boot_info. Also boot_info has depedency on mbedtls
so 'mbedtls' also added in zephyr module list for arcevent.

Change-Id: I01b293cc5f2358a16bb180d953132ba0913dc08e
Signed-off-by: Subrata Chatterjee <schatterjee@habana.ai>
---

diff --git a/package/zephyr/zephyr.mk b/package/zephyr/zephyr.mk
index 3c5e4a3..350a869 100644
--- a/package/zephyr/zephyr.mk
+++ b/package/zephyr/zephyr.mk
@@ -116,6 +116,8 @@
 ZEPHYR_CONFIGURE_CMDS += && $(ZEPHYR_CONF_ENV) $(HOST_DIR)/bin/cmake $(ZEPHYR_CONF_OPTS) $(ZEPHYR_CONF_OPTS_EVENT) -B $(ZEPHYR_EVENT_DIR)
 ZEPHYR_BUILD_CMDS += && $(TARGET_MAKE_ENV) $(MAKE) -C $(@D)/$(ZEPHYR_EVENT_DIR)
 ZEPHYR_INSTALL_TARGET_CMDS += && $(INSTALL) -D -m 0644 $(@D)/$(ZEPHYR_EVENT_DIR)/zephyr/zephyr.elf $(BINARIES_DIR)/zephyr_event.elf
+ZEPHYR_EVENT_MODULES = boot_info mbedtls
+$(foreach module,$(ZEPHYR_EVENT_MODULES),$(call DEFINE_MODULE_ENVVARS,$(module),$(ZEPHYR_EVENT_DIR),$(ZEPHYR_EVENT_NAME)))
 endif
 
 ifdef BR2_PACKAGE_ZEPHYR_ARC_PREBOOT
